build/DotnetCoreSdkLKG.props

```diff
@@ -1,31 +0,0 @@
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <CoreSetupRid>$(HostRid)</CoreSetupRid>
-    <CoreSetupRid Condition=" '$(HostOSName)' == 'win' or '$(HostOSName)' == 'osx' ">$(HostMonikerRid)</CoreSetupRid>
-
-    <!-- Use the portable linux-x64 Rid when downloading the shared framework compressed file.
-         NOTE: There isn't a 'linux-x64' version of the installers or the additional shared framweork. -->
-    <!-- Use the portable "linux-x64" Rid when downloading Linux shared framework compressed file. -->
-    <SharedFrameworkRid>$(CoreSetupRid)</SharedFrameworkRid>
-    <SharedFrameworkRid Condition=" '$(ProductMonikerRid)' == 'linux-musl-x64' ">$(ProductMonikerRid)</SharedFrameworkRid>
-    <SharedFrameworkRid Condition=" '$(UsePortableLinuxSharedFramework)' == 'true' ">linux-$(Architecture)</SharedFrameworkRid>
-
-    <DotNetCoreSdkLKGCompressedFileName>dotnet-sdk-$(DotNetCoreSdkLKGVersion)-$(SharedFrameworkRid)$(ArchiveExtension)</DotNetCoreSdkLKGCompressedFileName>
-  </PropertyGroup>
-
-  <PropertyGroup>
-    <DotNetCoreSdkLKGRootUrl>$(CoreSetupBlobRootUrl)Sdk/$(DotNetCoreSdkLKGVersion)/</DotNetCoreSdkLKGRootUrl>
```

Comment:
Delete declaration of `DotNetCoreSdkLKGVersion` and any other related properties.